20230319/mike.java

```diff
@@ -0,0 +1,67 @@
+/*
+Use TrieNode to solve it(which just implement few days ago).
+Because there are many possible, use recursive to solve.
+If meet '.', just seperate to some parts.
```

Comment:
gpt-refined description for your reference only.

```suggestion
To solve the problem, we use TrieNode, which we just implemented a few days ago.
Since there are many possible solutions, we use recursion to solve it.
If we encounter a ‘.’, we separate it into some parts.
```